templates/csi-daemonset.yaml

```
@@ -28,6 +28,7 @@ spec:
       {{ template "csi.pod.annotations" . }}
     spec:
       serviceAccountName: {{ template "vault.fullname" . }}-csi-provider
+      {{- template "csi.daemonSet.tolerations" . }}
```

Since tolerations are set here in the pod spec portion of the daemonSet, I'm thinking this might fit better in the `csi.pod` section.

```suggestion
      {{- template "csi.pod.tolerations" . }}
```